velox/functions/lib/window/tests/WindowTestBase.cpp

```diff
@@ -48,6 +49,81 @@ WindowTestBase::QueryInfo WindowTestBase::buildWindowQuery(
   return {op, functionSql, querySql};
 }
 
+namespace {
+void splitOrderBy(
+    const std::string& overClause,
+    vector_size_t startIdx,
+    vector_size_t length,
+    std::vector<std::string>& orderByClauses) {
+  auto orderByPars = overClause.substr(startIdx, length);
+  std::istringstream tokenStream(orderByPars);
+  std::string token;
+  while (std::getline(tokenStream, token, ',')) {
+    orderByClauses.push_back(token);
+  }
+}
+}; // namespace
+
+WindowTestBase::QueryInfo WindowTestBase::makeStreamingWindow(
+    const std::vector<RowVectorPtr>& input,
+    const std::string& function,
+    const std::string& overClause,
+    const std::string& frameClause) {
+  std::string functionSql =
+      fmt::format("{} over ({} {})", function, overClause, frameClause);
+  core::PlanNodePtr op = nullptr;
+  std::vector<std::string> orderByClauses;
+
+  std::string partitionByStr = "partition by";
```

Comment:
This and orderByStr can be static const strings and their variable names need to start with k